### PR TITLE
Update to poppler 0.32.0

### PR DESCRIPTION
**Pull request by svenax**
- Update to poppler 0.32.0
- Enable Qt5 support

### CODE REVIEW

**Comment by dunn** [comment]
#37280 is still open because a bunch of things that depend on poppler need to be re-built and there are a few obstacles I think.

File: Library/Formula/poppler.rb [hunk 1]
```diff
@@ -51,8 +53,12 @@ def install
 
     if build.with? "qt"
       args << "--enable-poppler-qt4"
+    elsif build.with? "qt5"
+      ENV.prepend_path "PKG_CONFIG_PATH", "#{Formula["qt5"].opt_prefix}/lib/pkgconfig"
```
Comment (DomT4) [inline Library/Formula/poppler.rb]: ` qt5 ` is ` keg_only ` so this should be done automatically - In theory.

Comment (svenax) [inline Library/Formula/poppler.rb]: Yup, but not in reality.

Comment (DomT4) [inline Library/Formula/poppler.rb]: Strange. Should be done by [this block](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/extend/ENV/super.rb#L134-L138).

Comment (svenax) [inline Library/Formula/poppler.rb]: It works for qt4 but not qt5 for some reason. I have not looked into the underlying reason for it, just added this line to get Poppler to build properly. I see several other formulas adding explicit PKG_CONFIG_PATH entries in a similar way.

**Comment by mikemcquaid** [comment]
Looks like the `dylib` may have changed in which case `brew uses poppler` stuff may need to have the `revision`s bumped/added: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/24217/version=yosemite/testReport/junit/brew-test-bot/yosemite/test_pdf2htmlex/

**Comment by jacknagel** [comment]
Closing as a dupe of #37280.

**Pull request closed without merging**